demo/grid/animated-grid.html

```diff
@@ -129,18 +130,16 @@
 
     },
 
-    listeners: {
-      click: '_onClick'
-    },
-
     _computeTileClass: function(color) {
       return 'tile ' + color + '-300';
     },
 
     _onClick: function(event) {
-      var target = event.target;
-      while (target !== this && !target._templateInstance) {
-        target = target.parentNode;
+      var target = event.composedPath ? event.composedPath()[0]
```

Comment:
is this conditional needed? You could do like in here (rootTarget thingy) https://github.com/PolymerElements/neon-animation/pull/206/files?diff=unified#diff-f58ce2895946f78bf1780674c5d9c23fR117